routes/adjustmentMethods/EToAdjustmentMethod.ts

```diff
@@ -0,0 +1,224 @@
+	const windSpeed = etoData.windSpeed / 2.237;
+
+	const avgTemp = ( maxTemp + minTemp ) / 2;
+
+	const saturationVaporPressureCurveSlope = 4098 * 0.6108 * Math.exp( 17.27 * avgTemp / ( avgTemp + 237.3 ) ) / Math.pow( avgTemp + 237.3, 2 );
+
+	const pressure = 101.3 * Math.pow( ( 293 - 0.0065 * elevation ) / 293, 5.26 );
+
+	const psychrometricConstant = 0.000665 * pressure;
+
+	const deltaTerm = saturationVaporPressureCurveSlope / ( saturationVaporPressureCurveSlope + psychrometricConstant * ( 1 + 0.34 * windSpeed ) );
+
+	const psiTerm = psychrometricConstant / ( saturationVaporPressureCurveSlope + psychrometricConstant * ( 1 + 0.34 * windSpeed ) );
+
+	const tempTerm = ( 900 / ( avgTemp + 273 ) ) * windSpeed;
+
+	const minSaturationVaporPressure = 0.6108 * Math.exp( 17.27 * minTemp / ( minTemp + 237.3 ) );
+
+	const maxSaturationVaporPressure = 0.6108 * Math.exp( 17.27 * maxTemp / ( maxTemp + 237.3 ) );
+
+	const avgSaturationVaporPressure = ( minSaturationVaporPressure + maxSaturationVaporPressure ) / 2;
+
+	const actualVaporPressure = ( minSaturationVaporPressure * etoData.maxHumidity / 100 + maxSaturationVaporPressure * etoData.minHumidity / 100 ) / 2;
+
+	const dayOfYear = moment.unix( etoData.timestamp ).dayOfYear();
+
+	const inverseRelativeEarthSunDistance = 1 + 0.033 * Math.cos( 2 * Math.PI / 365 * dayOfYear );
+
+	const solarDeclination = 0.409 * Math.sin( 2 * Math.PI / 365 * dayOfYear - 1.39 );
+
+	const latitudeRads = Math.PI / 180 * coordinates[ 0 ];
+
+	const sunsetHourAngle = Math.acos( -Math.tan( latitudeRads ) * Math.tan( solarDeclination ) );
+
+	const extraterrestrialRadiation = 24 * 60 / Math.PI * 0.082 * inverseRelativeEarthSunDistance * ( sunsetHourAngle * Math.sin( latitudeRads ) * Math.sin( solarDeclination ) + Math.cos( latitudeRads ) * Math.cos( solarDeclination ) * Math.sin( sunsetHourAngle ) );
+
+	const clearSkyRadiation = ( 0.75 + 2e-5 * elevation ) * extraterrestrialRadiation;
+
+	const solarRadiation = etoData.solarRadiation;
+
+	const netShortWaveRadiation = ( 1 - 0.23 ) * solarRadiation;
+
+	const netOutgoingLongWaveRadiation = 4.903e-9 * ( Math.pow( maxTemp + 273.16, 4 ) + Math.pow( minTemp + 273.16, 4 ) ) / 2 * ( 0.34 - 0.14 * Math.sqrt( actualVaporPressure ) ) * ( 1.35 * solarRadiation / clearSkyRadiation - 0.35);
+
+	const netRadiation = netShortWaveRadiation - netOutgoingLongWaveRadiation;
+
+	const radiationTerm = deltaTerm * 0.408 * netRadiation;
+
+	const windTerm = psiTerm * tempTerm * ( avgSaturationVaporPressure - actualVaporPressure );
+
+	return ( windTerm + radiationTerm ) / 25.4;
+}
+
+/**
+ * Approximates the wind speed at 2 meters using the wind speed measured at another height.
+ * @param speed The wind speed measured at the specified height (in miles per hour).
+ * @param height The height of the measurement (in feet).
+ * @returns The approximate wind speed at 2 meters (in miles per hour).
+ */
+export function standardizeWindSpeed( speed: number, height: number ) {
```

Comment:
It's not currently being used (it will be used when `getEToData` is implemented in WeatherProviders). Adding a dedicated utils file sounds like a great idea, but let's do that in a separate PR so we can move some other functions from weather.ts into there without changing the scope of this PR too much

Reply:
Fine by that. Resolving in favour of #64